catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -500,7 +501,7 @@ private void buildQueue(Stream<Path> ingestStream, ArrayBlockingQueue<Metacard>
                     List<Metacard> metacardList = zipDecompression.transform(inputStream,
                             arguments);
                     if (metacardList.size() != 0) {
-                        metacardFileMapping = generateFileMap(inputFile.getParentFile());
+                        metacardFileMapping = generateFileMap(new File(inputFile.getParent() + METACARD_PATH));
```

Comment:
Could you use: https://docs.oracle.com/javase/7/docs/api/java/io/File.html#File%28java.lang.String,%20java.lang.String%29